mod-data-export/src/test/resources/samples/auth_update_headings_failure.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "BULK_EDIT_QUERY",
+  "source": "folio",
+  "description": "BULK_EDIT_QUERY",
+  "isSystemSource": false,
+  "type": "BULK_EDIT_QUERY",
```

Comment:
Instead of separate file for failed type. You can just edit auth_update_headings_success.json.
`And set requestBodyOnSuccess.type = "BULK_EDIT_QUERY"`